src/main/java/org/wildfly/security/auth/provider/jdbc/JdbcSecurityRealmBuilder.java

```diff
@@ -0,0 +1,75 @@
+/*
+ * JBoss, Home of Professional Open Source
+ *
+ * Copyright 2013 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.wildfly.security.auth.provider.jdbc;
+
+import org.wildfly.security.auth.spi.RealmIdentity;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * <p>A builder class to that creates {@link JdbcSecurityRealm} instances.</p>
```

Comment:
This is on the link above but I'll state it here too: ``<p>`` is self-closing.  DocLint will throw an error if you try to add ``</p>``.  Also, it should be used between paragraphs, not before them.